mlflow/genai/labeling/labeling.py

```diff
@@ -46,63 +43,97 @@ class LabelingSession:
         `pip install mlflow[databricks]` to use it.
     """
 
-    def __init__(self, session: "_LabelingSession"):
-        self._session = session
+    def __init__(
+        self,
+        *,
+        name: str,
```

Comment:
I checked and verified that there are no call sites for the `LabelingSession` constructor from a Databricks Labeling Session within MLflow or known projects. This seems like a safe change.